src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const CORRECT_COUNT_WORKINGS = 3;
+  let callCount = 1;
+
+  return (firstNum) => (secondNum) => {
+    const isEven = callCount % 2 === 0;
+    const isBroken = callCount > CORRECT_COUNT_WORKINGS && isEven;
+
+    if (isBroken) {
+      callCount++;
+
+      return 'Bzzz... Error!';
+    }
+    callCount++;
```

Comment:
```suggestion
  let callCount = 0

  return (firstNum) => (secondNum) => {
    callCount++;

    const isEven = callCount % 2 === 0;
    const isBroken = callCount > CORRECT_COUNT_WORKINGS && isEven;

    if (isBroken) {
      return 'Bzzz... Error!';
    }
```
This way it is more obvious and there is no need for repeated `callCount++` statements.